packages/mui-base/src/Accordion/Item/AccordionItemContext.tsx

```diff
@@ -0,0 +1,30 @@
+'use client';
+import * as React from 'react';
+import type { AccordionItem } from './AccordionItem';
+
+export interface AccordionItemContext {
+  open: boolean;
+  ownerState: AccordionItem.OwnerState;
+  setTriggerId: (id: string | undefined) => void;
+  triggerId?: string;
+}
+/**
+ * @ignore - internal component.
```

Comment:
If you change the file extension to .ts (as there's no JSX here), you won't need to have this comment.